src/renderers/dom/ReactDOMServer.js

```diff
@@ -13,15 +13,15 @@
 
 var ReactDOMInjection = require('ReactDOMInjection');
 var ReactDOMStackInjection = require('ReactDOMStackInjection');
-var ReactServerRendering = require('ReactServerRendering');
+var ReactDOMServerRendering = require('ReactDOMServerRendering');
```

Comment:
Let's put this into another file so that we can get the roll out. `ReactDOMStreamServer`?